docs/security/parameters.md

```diff
@@ -0,0 +1,46 @@
+# Configuration
+
+To use PowSyBl OpenLoadFlow for all security analyses, you have to configure the `security-analysis` module in your configuration file:
+```yaml
+security-analysis:
+  default-impl-name: OpenLoadFlow
+```
+
+## Generic parameters
+
+Generic parameters are documented in [PowSyBl Core](inv:powsyblcore:*:*#simulation/security/configuration)
+
+The next section details the parameters that are specific to PowSyBl Open LoadFLow.
+
+## Specific parameters
+
+**contingencyPropagation**  
+The `contingencyPropagation` property:
```

Comment:
Maybe we can mention that contingency propagation is only relevant for node/breaker topology, no?

Reply:
In iidm topology is defined at the level of the voltage level, meaning that even if most of the time, all a network is of same topology but hybrid network is supported.

Reply:
improved this part